WordPress/gutenberg
# Fixes fixed block toolbar in widgets editor

## What?


Fixes #50367 

## Why?


Because the new position of the top toolbar in the widgets editor was not correct.

## How?


Style updates.

## Testing Instructions

1. Activate a classic theme
2. Go to Widgets
3. Activate top toolbar from the dot menu in the top right corner
4. Top toolbar should display correctly

### Testing Instructions for Keyboard


## Screenshots or screencast 



https://user-images.githubusercontent.com/107534/236435656-724bcf7e-c581-4552-a5ab-0b181237b666.mp4
